components/Header/index.jsx

```diff
@@ -0,0 +1,33 @@
+import Image from 'next/image';
+import { useState } from 'react';
+
+import Sidebar from '@/components/Sidebar';
+
+import logo from '@/public/logo-small.svg';
+
+const Header = ({ name, avatarUrl }) => {
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+
+  return (
+    <div className="flex justify-between mb-[42px]">
+      <div className="h-8 ml-8 mt-[34px]">
+        <Image src={logo} alt="logo" />
+      </div>
+      {avatarUrl && (
+        <div className="w-9 h-9 mt-8 mr-8 z-[1] cursor-pointer">
+          <Image
+            src={avatarUrl}
+            alt="profile avatar"
+            layout="fixed"
+            width={36}
+            height={36}
+            onClick={() => setIsSidebarOpen(value => !value)}
+          />
+        </div>
+      )}
```

Comment:
Should we have a default avatar URL? Otherwise it seems we cannot open the side bar (and so log out) 🤔

Reply:
that's interesting, you are getting an empty `avatar_url` as api response @malparty ?

Reply:
ha... I think I understand: a falsy `avatarUrl` means the user is not logged in, and so we do not display the side bar? If this assumption is correct, can we make this logic more explicit? 

I see 2 ways to make it more explicit: 
- `const isUserLoggedIn = (avatarUrl == false)` // And using `isUserLoggedIn` as a condition to display the side bar
- moving the SideBar into a function or another component, so that it can have the name "SideBar" (or "LoggedMenu")

It might be a little bit wordy, but that would make the code much easier to read later if future changes are needed ;-)

Reply:
I think we can get rid of this conditional rendering here altogether and move it up the component tree!
The purpose is to show the header and sidebar whenever the user is logged in, otherwise the user is redirected to `login` page, moved the condition up to the parent component in b091601745bf3988bdf30bf9e84b61bb25cc728f

Reply:
Ahh yes, it makes sense if we do not need this layout for the login page. Nice idea! 👍  And code is more explicit now 🥰